### PR TITLE
Added API calling functions:

### PR DESCRIPTION
**Pull request by HabibPayenda**
### The following requirements were added to the project:

Read the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) to learn how to use this API.
Create a new game with the name of your choice by using the API.
Make sure that you saved the ID of your game that will be returned by API.
Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
Use arrow functions instead of the function keyword.
Use async and await JavaScript features to consume the API.
No styling is required.

**Please check if I have missed something.**

**Thank you!**

### CODE REVIEW

File: src/modules/axiosHelper.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//     game = game.split(' ');
+//     // eslint-disable-next-line prefer-destructuring
+//     game = game[3];
+//     return game;
+//   } catch (error) {
+//     return error;
+//   }
+// };
+
+// const newGame = game();
```
Comment (prolajumokeoni) [inline src/modules/axiosHelper.js]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @HabibPayenda, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/axiosHelper.js (hunk 1 above)
Comment (HabibPayenda) [reply src/modules/axiosHelper.js]: @prolajumokeoni  thanks, I don't know how I forgot it.

**Review by addod19 — APPROVED** [review]
Hi @HabibPayenda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job removing the commented piece of code :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**